library/translation.inc.php

```diff
@@ -263,6 +263,15 @@ function html_header_show() {
     //  We commented this out since we are now standardizing encoding
     //  in the globals.php file.
```

Comment:
The function was planned to be a temporary solution.


A full blown AM will replace all resource loading in all HTML headers.



--ef.-1.c0046808818e57e33cd3f822a6847dda.ef--